Sources/KlaviyoSwift/KlaviyoAPI.swift

```diff
@@ -49,6 +49,7 @@ struct KlaviyoAPI {
             requestFailed(request, error, 0.0)
             return .failure(.internalRequestError(error))
         }
+        urlRequest.allHTTPHeaderFields?["X-Klaviyo-Retry-Attempt"] = "\(attemptNumber)/50"
```

Comment:
maybe could use a constant here.

Reply:
Just to be super sure, this header should be formatted `1/50` -> `50/50` ?

Reply:
yah that was my understanding...could totally have missed that detail though!

Reply:
Makes sense to me, just want to be sure we don't accidentally start at 0 (or end at 49)

Reply:
Yah I'm going to test it through and ensure it does the desired thing.

Reply:
I wonder if the backend could be able to reply with a header to tell you to increase your max attempts.

Reply:
@jason-myers-klaviyo let us know if there are specific requirements but IIRC you wanted it to send the last request with 50/50 right?

Reply:
Yes, I think that's right.  If we see a 50/50 and we return a 429, we'll know that the request will not be tried again.

50/50 meaning "we're on attempt number 50, and we will stop after 50 (meaning we won't go again after this one"

And 1/50 meaning "we're on the first attempt of 50 possible attempts"

Reply:
Lets name this header `X-Klaviyo-Attempt-Count` then?

Reply:
I think we landed on

`X-Klaviyo-Retry-Attempt`

Earlier

Reply:
Yeah its a nitpick, but just responding to the above thread, that name is a little ambiguous: is `1/50` the first _retry_ or first _attempt_.

Reply:
I see, no issues here then

Reply:
Starting from 1 now. Still doing some testing on my end but so far it's looking good.

Also, renamed header to `X-Klaviyo-Attempt-Count`